.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
In general layout looks good, really nice animation of card movement from deck to additional open cards.